scripts/setup/debians/build_install_debians.sh

```diff
@@ -43,26 +43,55 @@ elif [ "$DIST" = "bionic" ]; then
   sudo dpkg -i libopencv*_amd64.deb || exit 1
 
   # alvar
-  cp ${DEBIAN_LOC}/files_18_04/alvar_rules ${DEBIAN_LOC}/alvar/rules
-  cp ${DEBIAN_LOC}/files_18_04/alvar_control ${DEBIAN_LOC}/alvar/control
-  cp ${DEBIAN_LOC}/files_18_04/alvar_changelog ${DEBIAN_LOC}/alvar/changelog
+  cp ${DEBIAN_LOC}/files/alvar_rules ${DEBIAN_LOC}/alvar/rules
+  cp ${DEBIAN_LOC}/files/alvar_control ${DEBIAN_LOC}/alvar/control
+  cp ${DEBIAN_LOC}/files/alvar_changelog ${DEBIAN_LOC}/alvar/changelog
   # dlib
-  cp ${DEBIAN_LOC}/files_18_04/dlib_rules ${DEBIAN_LOC}/dlib/rules
-  cp ${DEBIAN_LOC}/files_18_04/dlib_control ${DEBIAN_LOC}/dlib/control
-  cp ${DEBIAN_LOC}/files_18_04/dlib_changelog ${DEBIAN_LOC}/dlib/changelog
+  cp ${DEBIAN_LOC}/files/dlib_rules ${DEBIAN_LOC}/dlib/rules
+  cp ${DEBIAN_LOC}/files/dlib_control ${DEBIAN_LOC}/dlib/control
+  cp ${DEBIAN_LOC}/files/dlib_changelog ${DEBIAN_LOC}/dlib/changelog
   # dbow2
-  cp ${DEBIAN_LOC}/files_18_04/dbow2_rules ${DEBIAN_LOC}/dbow2/rules
-  cp ${DEBIAN_LOC}/files_18_04/dbow2_control ${DEBIAN_LOC}/dbow2/control
-  cp ${DEBIAN_LOC}/files_18_04/dbow2_changelog ${DEBIAN_LOC}/dbow2/changelog
+  cp ${DEBIAN_LOC}/files/dbow2_rules ${DEBIAN_LOC}/dbow2/rules
+  cp ${DEBIAN_LOC}/files/dbow2_control ${DEBIAN_LOC}/dbow2/control
+  cp ${DEBIAN_LOC}/files/dbow2_changelog ${DEBIAN_LOC}/dbow2/changelog
   # gtsam
-  cp ${DEBIAN_LOC}/files_18_04/gtsam_changelog ${DEBIAN_LOC}/gtsam/changelog
+  cp ${DEBIAN_LOC}/files/gtsam_changelog ${DEBIAN_LOC}/gtsam/changelog
   # decomputil
   # jps3d
   sudo apt-get install -y libvtk6.3 libboost-filesystem1.62.0 libboost-system1.62.0
-  cp ${DEBIAN_LOC}/files_18_04/jps3d_changelog ${DEBIAN_LOC}/jps3d/changelog
+  cp ${DEBIAN_LOC}/files/jps3d_changelog ${DEBIAN_LOC}/jps3d/changelog
   # openmvg
 elif [ "$DIST" = "focal" ]; then
   echo "Ubuntu 20 detected"
+  # Install dependencies
+
+  # opencv
+  cd ${DEBIAN_LOC}/opencv
+  sudo mk-build-deps -i -r -t "apt-get --no-install-recommends -y" control
+  cd ${DEBIAN_LOC}
+  ./build_opencv.sh || exit 1
+  cd ${DEBIAN_LOC}/libopencv
+  sudo dpkg -i libopencv*_amd64.deb || exit 1
+
+  # alvar
```

Comment:
it seems like there is a lot of overlap between these two ubuntu versions... can we make them share more of the script?